pkgpanda/docs/http-swagger.yaml

```diff
@@ -0,0 +1,116 @@
+swagger: '2.0'
+info:
+  title: DC/OS Pkgpanda API
+  version: '1.0.0'
+basePath: /pkgpanda
+
+
+definitions:
+
+  PackageId:
+    description: A unique pkgpanda package ID. Composed of a name and a version separated by `--`.
+    type: string
+    example: 'mesos--abcdef'
+    # Pattern adapted from pkgpanda/__init__.py.
+    pattern: '/^[a-zA-Z0-9@_+]([a-zA-Z0-9@._+\-]*[a-zA-Z0-9@._+])?--[a-zA-Z0-9@_+:.]+$/'
+
+  PackageIdArray:
+    description: An array of unique pkgpanda package IDs.
+    type: array
+    uniqueItems: true
+    items:
+      $ref: '#/definitions/PackageId'
+
+
+parameters:
+
+  PackageId:
+    name: package-id
+    in: path
+    required: true
+    # From #/definitions/PackageId.
+    description: A unique pkgpanda package ID. Composed of a name and a version separated by `--`.
+    type: string
+    pattern: '/^[a-zA-Z0-9@_+]([a-zA-Z0-9@._+\-]*[a-zA-Z0-9@._+])?--[a-zA-Z0-9@_+:.]+$/'
+
+
+paths:
+
+  /:
+    get:
+      summary: List the packages that are present in the node's pkgpanda repository.
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: A list of the packages on this node.
+          schema:
+            '$ref': '#/definitions/PackageIdArray'
+
+  /{package-id}:
+    get:
+      summary: Get a package in the node's pkgpanda repository.
```

Comment:
This endpoint responds with a 204 No Content on success, so that's not accurate.